be/src/storage/lake/update_manager.cpp

```diff
@@ -73,6 +73,10 @@ Status UpdateManager::publish_primary_key_tablet(const TxnLogPB_OpWrite& op_writ
     RETURN_IF_ERROR(state.rewrite_segment(op_write, metadata, tablet));
     cost_str << " [UpdateStateCache rewrite segment] " << watch.elapsed_time();
     watch.reset();
+    std::vector<ColumnUniquePtr> auto_increment_deletes;
+    if (op_write.has_txn_meta() && op_write.txn_meta().has_auto_increment_partial_update_column_id()) {
+        state.fetch_conflict_deletes_by_auto_increment(op_write.rowset().segments_size(), auto_increment_deletes);
```

Comment:
Can `fetch_conflict_deletes_by_auto_increment` hide in `state->load`?

Reply:
done